src/core/index.ts

```diff
@@ -75,16 +96,39 @@ interface BellPromptConfig {
     };
     colors?: {
         bell?: {
+            fg?: string;
             bg?: string;
+        };
+    };
+}
+
+export interface AlertPromptConfig extends BasePromptConfig {
+    type: 'alert';
+    trigger: PromptTrigger;
```

Comment:
Do we need to declare trigger again, would it be inherited from base?

Reply:
Ahh nope, will remove